src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/SocketClientLoggerExtensions.cs

```diff
@@ -153,6 +153,8 @@ internal static class SocketClientLoggerExtensions
         private static readonly Action<ILogger, DateTime, string, string, Exception> _exceptionThrownFromCallback =
             LoggerMessage.Define<DateTime, string, string>(LogLevel.Error, new EventId(19, nameof(ExceptionThrownFromCallback)), "{time}: Connection Id {connectionId}: An exception was thrown from the '{callback}' callback");
```

Comment:
Unrelated to this PR, but could you add a `.` to this log?

Reply:
Fixed.